src/boot/bootctl.c

```diff
@@ -164,6 +163,8 @@ static int load_install_machine_id_and_layout(void) {
                 if (r < 0 && !IN_SET(r, -ENOENT, -ENOMEDIUM))
                         return log_error_errno(r, "Failed to get machine-id: %m");
         } else {
+                log_notice("Read kernel install machine ID from /etc/machine-info. Please migrate to /etc/kernel/entry-token instead.");
```

Comment:
I'll simplify it further to: "Read $KERNEL_INSTALL_MACHINE_ID from /etc/machine-info. Please move it to /etc/kernel/entry-token."

i.e. I dropped "token from". Why? because we actually only parse $KERNEL_INSTALL_MACHINE_ID is as 128bit id, and don't allow anything else. Which is a bit different for the entry token which can be any string. Hence using the same word for the data from $KERNEL_INSTALL_MACHINE_ID and /etc/kernel/entry-token might be slightly misleading, given it's different structure. But if we just drop "token" from the sentence this goes away and the sentence still makes sense.